x-pack/plugins/index_management/public/application/sections/home/data_stream_list/data_stream_table/data_stream_table.tsx

```diff
@@ -152,7 +152,10 @@ export const DataStreamTable: React.FunctionComponent<Props> = ({
     name: (
       <EuiToolTip
         content={i18n.translate('xpack.idxMgmt.dataStreamList.table.dataRetentionColumnTooltip', {
-          defaultMessage: `Data is kept at least this long before being automatically deleted. The data retention value only applies to the data managed directly by the data stream. If some data is subject to an index lifecycle management policy, then the data retention value set for the data stream doesn't apply to that data.`,
+          defaultMessage: `Data is kept at least this long before being automatically deleted. The data retention value only applies to the data managed directly by the data stream. {canEnableDataRetention, plural, one {If some data is subject to an index lifecycle management policy, then the data retention value set for the data stream doesn't apply to that data.} other {}}`,
+          values: {
+            canEnableDataRetention: config.enableTogglingDataRetention ? 1 : 0,
```

Comment:
nit: maybe `canDisableDataRetention` would be a more suitable name for the variable? Also, can we add a short comment explaining why we exclude this sentence when the user can't disable data retention?